test/tools/isoltest.cpp

```diff
@@ -242,8 +246,31 @@ TestStats TestTool::processPath(
 
 }
 
+/// Ensures the terminal is set up properly.
```

Comment:
Not sure whether this comment provides more information than the name of the function.